src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetTracking.kt

```diff
@@ -7,19 +7,23 @@ class WidgetTracking {
 
     class Categories {
         val widget = "Widget"
+        val abtesting = "abtesting"
     }
 
     class Actions {
         val loaded = "loaded"
         val buttonPressed = "buttonPressed"
         val error = "error"
+        val abtest = "MitigateFallback"
     }
 
     class Names {
         val widget = "widget"
         val startIdent = "startIdent"
         val incompatible = "unsupportedOS"
         val fallback = "fallback"
+        val abtestOriginal = "original"
+        val abtestVariation1 = "Variation1"
```

Comment:
I personally prefer to name the variation that reflects the change, when we change the blue button, e.g. `variation:button:blue` or rather if project-wise `variation:improve-cta-button`, so when we do multiple tests, it would be easier to identify in the UI

Reply:
also, should it be `variation1`, or why the first letter is uppercased?

Reply:
Well, it's not like only *one* thing changed, but a lot. That's why I considered the name ok-ish. The name "Variation1" was created in Matomo like that (also with capital first letter, don't know why...), that's why it's like this here, too.